src/opnsense/mvc/app/models/OPNsense/Interfaces/FieldTypes/LinkAddressField.php

```diff
@@ -113,10 +113,13 @@ public function getValidators()
      */
     public function getDescription()
     {
-        if (isset(self::$known_addresses[$this->internalValue])) {
-            return self::$known_addresses[$this->internalValue];
+        $value = (string)$this;
```

Comment:
we could make this explicit, since it relies on `__toString()` to work the magic here:

```suggestion
        $value = $this-->toString();
```

Reply:
@fichtner we can use `$this->getCurrentValue()`, the string cast seemed to be missing indeed.

Reply:
Ah, ok, shouldn't basefield __toString also use getCurrentValue() then? Right now we need to override __toString and getCurrentValue because the current value is not internalValue :/

Reply:
(feel free to take this over, would probably take longer for me)